src/app/about/people/page.js

```diff
@@ -0,0 +1,43 @@
+import Card from "../../components/Card"
+
+import nn from "../../images/officers/nn.jpg"
+import ks from "../../images/officers/ks.jpg"
+import jb from "../../images/officers/jb.jpg"
+import am from "../../images/officers/am.jpg"
+import bm from "../../images/officers/bm.jpg"
+import kds from "../../images/officers/kds.jpg"
+
+const OurTeam = () =>
+{
+ return (
+  <div className="section section-team">
+
+   <div className="text-center mt-[2rem]">
+
+    <h1 className="font-bold text-[3rem] md:text-[3.5rem] lg:text-[4rem]">Our <div class="red-bold">People</div></h1>
+   </div>
+
+
+
+
+
+   <h3 className="text-black text-[3rem] font-semibold text-center mt-[5rem] p-0"> 2024 - 2025 </h3>
+   <h3 className="text-black text-[3rem] font-bold text-center">Executive Board</h3>
+
+
+
+
+   <div id="grid">
```

Comment:
The height of the card and the styling of the name and position seems to have changed, potentially from the Tailwind CSS migration. 
![image](https://github.com/user-attachments/assets/9ee77cc6-0e33-496c-a0f2-7851adef0b7a)

It should look similar to 
![image](https://github.com/user-attachments/assets/efc08990-3066-4470-b56c-a468261b166f)


Thank you!

Reply:
yeah I noticed that too. Should be an easy fix.